rdp-server/peer.c

```diff
@@ -980,7 +980,13 @@ static int pre_connect_handler(int mask, int fd, HANDLE handle, void *data) {
 		}
 
 		/* we have the magic bytes, let's read the RDPEPS blob */
-		read(peer->sockfd, pre_blob, sizeof(pre_blob_magic)); /* skip the pre blob magic */
+		status = read(peer->sockfd, pre_blob,
```

Comment:
I understand the logic to please the compiler with formal checks, anyway perhaps we should complain super loudly (something more alarmous than just `error skipping pre blob magic`) when we have peek bytes from the socket, and when really reading them, we figure it's something different...

Reply:
did continue the style like any other check before or after is done. (message wise)